tests/components/zwave_js/conftest.py

```diff
@@ -687,9 +687,17 @@ async def disconnect():
 
         client.version = VersionInfo.from_message(version_state)
         client.ws_server_url = "ws://test:3000/zjs"
+
+        def has_device_config_changed_side_effect(message, require_schema=None):
```

Comment:
```suggestion
        async def async_send_command_side_effect(message, require_schema=None):
```

Reply:
async.... 🤦🏾

Reply:
nope still not working

Reply:
OK figured out a large chunk of it, will update soon